tests/foreman/cli/test_leapp_client.py

```diff
@@ -0,0 +1,375 @@
+        'name': f'Red Hat Enterprise Linux 8 for x86_64 - BaseOS RPMs {RHEL8_VER}',
+        'releasever': RHEL8_VER,
+        'reposet': 'Red Hat Enterprise Linux 8 for x86_64 - BaseOS (RPMs)',
+    },
+    'rhel8_aps': {
+        'id': 'rhel-8-for-x86_64-appstream-rpms',
+        'name': f'Red Hat Enterprise Linux 8 for x86_64 - AppStream RPMs {RHEL8_VER}',
+        'releasever': RHEL8_VER,
+        'reposet': 'Red Hat Enterprise Linux 8 for x86_64 - AppStream (RPMs)',
+    },
+    'rhel9_bos': {
+        'id': 'rhel-9-for-x86_64-baseos-rpms',
+        'name': f'Red Hat Enterprise Linux 9 for x86_64 - BaseOS RPMs {RHEL9_VER}',
+        'releasever': RHEL9_VER,
+        'reposet': 'Red Hat Enterprise Linux 9 for x86_64 - BaseOS (RPMs)',
+    },
+    'rhel9_aps': {
+        'id': 'rhel-9-for-x86_64-appstream-rpms',
+        'name': f'Red Hat Enterprise Linux 9 for x86_64 - AppStream RPMs {RHEL9_VER}',
+        'releasever': RHEL9_VER,
+        'reposet': 'Red Hat Enterprise Linux 9 for x86_64 - AppStream (RPMs)',
+    },
+}
+
+
+@pytest.fixture
+def function_leapp_cv(module_target_sat, module_sca_manifest_org):
+    logger.info('Creating Leapp Content View')
+    return module_target_sat.api.ContentView(organization=module_sca_manifest_org).create()
+
+
+@pytest.fixture(scope='module')
+def module_leapp_lce(module_target_sat, module_sca_manifest_org):
+    logger.info('Creating Leapp Lifecycle Environment')
+    return module_target_sat.api.LifecycleEnvironment(organization=module_sca_manifest_org).create()
+
+
+@pytest.fixture
+def function_leapp_ak(
+    module_target_sat, function_leapp_cv, module_leapp_lce, module_sca_manifest_org, upgrade_path
+):
+    logger.info(
+        'Creating Leapp Activation Key for RHEL %s -> %s',
+        upgrade_path['source_version'],
+        upgrade_path['target_version'],
+    )
+    # need to publish and promote cv on lce to get content available on client host
+    function_leapp_cv.publish()
+    cvv = function_leapp_cv.read().version[0]
+    cvv.promote(data={'environment_ids': module_leapp_lce.id, 'force': True})
+    function_leapp_cv = function_leapp_cv.read()
+    ak = module_target_sat.api.ActivationKey(
+        content_view=function_leapp_cv,
+        environment=module_leapp_lce,
+        organization=module_sca_manifest_org,
+    ).create()
+    # In case of 'rhel7_server_extras' repos, need to enabled (overridden) repository
+    # to get content available on client host
+    if upgrade_path['source_version'].split('.')[0] == '7':
+        ActivationKey.content_override(
+            {
+                'id': ak.id,
+                'content-label': RHEL_REPOS['rhel7_server_extras']['id'],
+                'value': 'true',
+            }
+        )
+    return ak
+
+
+@pytest.fixture
+def verify_target_repo_on_satellite(
+    module_target_sat,
+    function_leapp_cv,
+    module_sca_manifest_org,
+    module_leapp_lce,
+    upgrade_path,
+    function_leapp_ak,
```

Comment:
You are not using this fixture in the function. Can you either remove it or use `@pytest.mark.usefixtures('function_leapp_ak')`?

Reply:
I would insist on removing this fixture if the test is not using it. Let me know if you hit any issues after its removal.